src/v2i-hub/ImmediateForwardPlugin/src/ImmediateForwardPlugin.h

```diff
@@ -55,6 +55,7 @@ class ImmediateForwardPlugin : public tmx::utils::PluginClient
 
 		// Mutex along with the data it protects.
 		std::mutex _mutexUdpClient;
+		// A vector of UDP clients for recieving V2X communication from different RSUs
```

Comment:
Sorry, I might misunderstood the functionality involved in this UDP client. Is it receiving message or sending to RSUs?

Reply:
No you are right. It is sending information to RSUs. I will correct the comment